src/test/java/org/cactoos/iterator/IteratorOfTest.java

```diff
@@ -60,7 +62,12 @@ public void emptyIteratorThrowsException() {
 
     @Test
     public void nonEmptyIteratorDoesNotHaveNext() {
-        final IteratorOf<Integer> iterator = this.iteratorWithFetchedElements();
+        final Iterator<Integer> iterator = new Skipped<>(
```

Comment:
@fanifieiev please do not rely on `Skipped` here, it's too far away from the tested class to make the test robust